### ISSUE TITLE
Failing travis builds

### ISSUE THREAD

**Issue by jchodera** [choderalab/bayesian-itc#81]
@bas-rustenburg : Can you take a look at the failing travis builds?

https://travis-ci.org/choderalab/bayesian-itc?utm_source=email&utm_medium=notification

Looks like Python 3.5 is failing:
```
======================================================================
ERROR: Test the importing of all libraries.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/choderalab/bayesian-itc/bitc/tests/test_bitc.py", line 15, in test_imports
    self._imports('bitc.' + mod)
  File "/home/travis/build/choderalab/bayesian-itc/bitc/tests/test_bitc.py", line 9, in _imports
    self.assertIsInstance(__import__(mod), ModuleType)
  File "/home/travis/build/choderalab/bayesian-itc/bitc/models.py", line 11, in <module>
    import pymc
  File "/home/travis/miniconda3/envs/test/lib/python3.5/site-packages/pymc/__init__.py", line 30, in <module>
    from .CommonDeterministics import *
  File "/home/travis/miniconda3/envs/test/lib/python3.5/site-packages/pymc/CommonDeterministics.py", line 21, in <module>
    from .utils import safe_len, stukel_logit, stukel_invlogit, logit, invlogit, value, find_element
  File "/home/travis/miniconda3/envs/test/lib/python3.5/site-packages/pymc/utils.py", line 14, in <module>
    from . import flib
ImportError: libopenblas.so.0: cannot open shared object file: No such file or directory
----------------------------------------------------------------------
```
It looks like the `.travis.yml` is out of date. Can you update it to
* Use `omnia` and `conda-forge`
* Test 2.7, 3.5, 3.6

**Comment by bas-rustenburg** [comment]
Added to my todo list for the week.

**Comment by bas-rustenburg** [comment]
Fixed in #82.
